### PR TITLE
Apply general code formatting to some pages in the tutorial section

### PR DESCRIPTION
**Pull request by YeldhamDev**
Changes include:

- Make it follow the official style guidelines.
- Use more Python-like syntax (`not` instead of `!`).
- Fix small indentation mistakes.
- Comment formatting (capitalization, end period).

### CODE REVIEW

File: tutorials/io/background_loading.rst [hunk 1]
```diff
@@ -181,12 +183,14 @@ loader.
         # Update your progress bar?
         get_node("progress").set_progress(progress)
 
-        # ... or update a progress animation?
+        # ...or update a progress animation?
```
Comment (akien-mga) [inline tutorials/io/background_loading.rst:186]: Not sure about this change, `... continuation` seems more correct to me.

Comment (akien-mga) [reply tutorials/io/background_loading.rst:186]: I looked a few online resources on style of ellipses, and dang, it's far from clear nor consensual... Apparently some style guides even want it styled as ` . . . ` (with spaces everywhere)...

Comment (Calinou) [reply tutorials/io/background_loading.rst:186]: I write it `...like this` myself.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**